scripts/postbuild.js

```diff
@@ -0,0 +1,26 @@
+var fs = require('fs');
+
+// Ensure that any optional peer dependency (e.g. ReactTransitionGroup and
+// ReactPopper) are properly set as optional globals. Specifically, do not
+// error if the user has not loaded these dependencies. Degrade gracefully.
+
+fs.readFile(__dirname + '/../dist/reactstrap.min.js', 'utf8', function(err, data) {
+  if (err) {
+    return console.error(err);
+  }
+
+  var result = data.replace(
+    /\,([a-zA-Z]+)\.ReactTransitionGroup\.Transition/,
```

Comment:
Should this and the one below have the global flag?

Reply:
Nope, just want to replace the top occurrence (it's only included once); however, to be sure we don't brick anything else only the factory load -- once.